x-pack/plugins/security_solution/public/common/components/event_details/osquery_tab.tsx

```diff
@@ -47,7 +47,7 @@ export const useOsqueryTab = ({
     : undefined;
 
   const responseActions =
-    expandedEventFieldsObject?.kibana?.alert?.rule?.parameters?.[0].response_actions;
+    expandedEventFieldsObject?.kibana?.alert?.rule?.parameters?.[0]?.response_actions;
```

Comment:
I found this error when fixing Cypress tests for guided onboarding tour
<img width="2560" alt="Screenshot 2024-04-22 at 12 48 12" src="https://github.com/elastic/kibana/assets/6295984/fede5add-5fa8-4543-8e90-320d37b338bb">